src/khoj/routers/web_client.py

```diff
@@ -199,115 +135,23 @@ def agent_page(request: Request, agent_slug: str):
         "creator_not_self": agent.creator != user,
     }
 
-    return templates.TemplateResponse(
-        "agent.html",
-        context={
-            "request": request,
-            "agent": agent_metadata,
-            "khoj_version": state.khoj_version,
-            "username": user.username if user else None,
-            "has_documents": has_documents,
-            "is_active": has_required_scope(request, ["premium"]),
-            "user_photo": user_picture,
-        },
-    )
+    return templates.TemplateResponse("agent.html", context=user_config)
 
 
-@web_client.get("/config", response_class=HTMLResponse)
+@web_client.get("/configure", response_class=HTMLResponse)
```

Comment:
Relevant code for renaming the page to `settings`.

```suggestion
@web_client.get("/settings", response_class=HTMLResponse)
```